### PR TITLE
Docs change for Zowe Explorer plug-in for IntelliJ IDEA

### PR DESCRIPTION
**Pull request by KUGDev**
Describe your pull request here:
Changes for Zowe Explorer plug-in for IntelliJ IDEA and Zowe Client Kotlin SDK

List the file(s) included in this PR:
See the files, included in the PR

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Comment by zFernand0** [comment]
The build should be fixed on the `v3-doc-branch` now. Feel free to update your branch 🥳

**Comment by KUGDev** [comment]
> The build should be fixed on the `v3-doc-branch` now. Feel free to update your branch 🥳

Did rebase
@anaxceron

**Review by anaxceron — COMMENTED** [review]
Left some comments re: style, grammar, etc.

File: docs/getting-started/overview.md [hunk 1]
```diff
@@ -268,21 +269,21 @@ ZEBRA Provides re-usable and industry compliant JSON formatted RMF/SMF data reco
...
+- Organize datasets on z/OS, files on USS into working sets
+- Allocate datasets, create members, files and directories with different permissions
+- Perform operations like renaming, copying and moving data in a modern way
+- Edit datasets, files and members. Smart auto-save keeps your content both in the editor and on the mainframe in-sync
+- Create multiple connections to different z/OS systems
+- Perform all available operations with jobs
+- Work with TSO Console directly in the IDE
 
-For more information, see [Using Zowe IntelliJ plug-in](../user-guide/intellij-using.md).
+To see more about it, you can start with [Zowe Explorer plug-in for IntelliJ IDEA use cases](../user-guide/intellij-use-cases.md).
```
[the remark this answers is not in the document]
Comment (KUGDev) [reply docs/getting-started/overview.md:286]: Why not?:)

File: docs/getting-started/zowe_faq.md [hunk 2]
```diff
@@ -521,42 +521,32 @@ Install the plug-in in your IntelliJ-based IDE directly from marketplace or down
...
 
-### How do I get help with using Zowe IntelliJ plug-in?
+### How do I get help with using Zowe Explorer plug-in for IntelliJ IDEA?
 
 <details className="zowe-faq">
 
 <summary></summary>
 
-You can read detailed user guide and find any information you need [here](https://plugins.jetbrains.com/plugin/18688-zowe-explorer/user-guide). Also, you can ask any questions in the Zowe Slack channel [#zowe-explorer-intellij](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
+You can start with the [use cases](../user-guide/intellij-use-cases.md) section of the usage guide and continue with all the other related sections that describe in details how to get started. Also, you can ask any questions in our [Zowe Slack channel (#zowe-explorer-intellij)](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
```
[the remark this answers is not in the document]
Comment (KUGDev) [reply docs/getting-started/zowe_faq.md:540]: There is no "user guide" as a singular section

File: docs/getting-started/zowe_faq.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Zowe technology can be used by a variety of mainframe IT and non-IT professional
 
 <summary></summary>
 
-Zowe consists of several components. The primary languages for API Mediation Layer are Java and JavaScript. Zowe CLI, Explorer for VSCode and Desktop are written in TypeScript. Explorer for IntelliJ is written in Kotlin, ZSS is written in C, while the cross memory server is written in metal C.
+Zowe consists of several components. The primary languages for API Mediation Layer are Java and JavaScript. Zowe CLI, Explorer for VSCode and Desktop are written in TypeScript. Zowe Explorer plug-in for IntelliJ IDEA is written in Kotlin, ZSS is written in C, while the cross memory server is written in metal C.
```
Comment (anaxceron) [inline docs/getting-started/zowe_faq.md:37]: Zowe Docs uses the Oxford comma: "Zowe CLI, Explorer for VS Code, and Desktop ..."

File: docs/getting-started/zowe_faq.md [hunk 4]
```diff
@@ -499,19 +499,19 @@ As a developer, you may contribute to Zowe Explorer in the following ways:
 
 </details>
 
-## Zowe IntelliJ plug-in FAQ
+## Zowe Explorer plug-in for IntelliJ IDEA FAQ
 
-### Why might I use Zowe IntelliJ plug-in versus a traditional ISPF interface to perform mainframe tasks?
+### Why might I use the plug-in versus a traditional ISPF interface to perform mainframe tasks?
```
Comment (anaxceron) [inline docs/getting-started/zowe_faq.md:504]: I think we should change the question to "Why **should** I use ... " or "Why **would** I use ... "

File: docs/getting-started/zowe_faq.md [hunk 5]
```diff
@@ -521,42 +521,32 @@ Install the plug-in in your IntelliJ-based IDE directly from marketplace or down
...
 
-### How do I get help with using Zowe IntelliJ plug-in?
+### How do I get help with using Zowe Explorer plug-in for IntelliJ IDEA?
 
 <details className="zowe-faq">
 
 <summary></summary>
 
-You can read detailed user guide and find any information you need [here](https://plugins.jetbrains.com/plugin/18688-zowe-explorer/user-guide). Also, you can ask any questions in the Zowe Slack channel [#zowe-explorer-intellij](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
+You can start with [use cases](../user-guide/intellij-use-cases.md) section to learn about use cases and to learn how to install and use the plug-in. Also, you can ask any questions in our [Zowe Slack channel (#zowe-explorer-intellij)](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
```
Comment (anaxceron) [inline docs/getting-started/zowe_faq.md:540]: missing a word; capitalize the section title; remove one unnecessary "to learn": "You can start with **the** **U**se case section to learn about use case and how to install and ... "

**Review by anaxceron — APPROVED** [review]
Made additional comments re: style, typos, etc. Please fix before merging the PR.

File: docs/getting-started/install-zowe-sdks.md [hunk 6]
```diff
@@ -55,38 +55,7 @@ implementation group: 'org.zowe.client.java.sdk', name: 'zowe-client-java-sdk',
...
-```
-implementation("org.zowe.sdk:zowe-kotlin-sdk:$version")
-```
-
-To use:
-
-```
-import org.zowe.kotlinsdk
-```
+For the detailed information about how to install the library, refer to [the official installation guide for Zowe Client Kotlin SDK](https://zowe.github.io/zowe-client-kotlin-sdk/#installation)
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/getting-started/install-zowe-sdks.md]: @KUGDev -- since this wasn't corrected, I had to modify this myself.

**Pull request merged**